### PR TITLE
import DNX project types

### PR DESCRIPTION
**Pull request by brentschmaltz**
Fixup project for dnx.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115%23discussion_r28101323%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115%23discussion_r28104193%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115%23discussion_r28107269%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20d3562b29ac165ea7790bc90515d0f9ba22b48f03%20src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115%23discussion_r28101323%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Do%20we%20need%20this%20check%20here%3F%20Few%20lines%20below%20this%20line%20of%20code%2C%20we%20check%20for%20the%20null%20value.%20Does%20it%20have%20any%20perf%20benefit%20checking%20it%20here%3F%20If%20so%2C%20we%20should%20remove%20the%20check%20from%20the%20other%20line.%22%2C%20%22created_at%22%3A%20%222015-04-09T20%3A51%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%2C%20%7B%22body%22%3A%20%22Agreed%20updated.%22%2C%20%22created_at%22%3A%20%222015-04-09T21%3A25%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3172421%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/brentschmaltz%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-04-09T22%3A02%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs%3AL96-103%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull d3562b29ac165ea7790bc90515d0f9ba22b48f03 src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs 5'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115#discussion_r28101323'>File: src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs:L96-103</a></b>
- <a href='https://github.com/tushargupta51'></a> Do we need this check here? Few lines below this line of code, we check for the null value. Does it have any perf benefit checking it here? If so, we should remove the check from the other line.
- <a href='https://github.com/brentschmaltz'></a> Agreed updated.

<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115?approve=1'></a>&nbsp;<a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by brandwe** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/115'>Refresh</a>.<a href='#crh-update-none'></a>

File: src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs [hunk 1]
```diff
@@ -96,7 +96,7 @@ public OpenIdConnectMessage(IEnumerable<KeyValuePair<string, string[]>> paramete
 
             foreach (KeyValuePair<string, string[]> keyValue in parameters)
             {
-                if (keyValue.Value != null)
+                if (keyValue.Value != null && !string.IsNullOrWhiteSpace(keyValue.Key))
```
Comment (tushargupta51) [inline src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs]: Do we need this check here? Few lines below this line of code, we check for the null value. Does it have any perf benefit checking it here? If so, we should remove the check from the other line.

Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs]: Agreed updated.

Comment (tushargupta51) [inline src/Microsoft.IdentityModel.Protocol.Extensions/OpenIdConnectMessage.cs]: :+1:

**Pull request merged**